Lesson21/Kadir/app.js

```diff
@@ -17,35 +17,64 @@ window.addEventListener('load',function(){
   // Show images function
   function showImage(index) {
     // Remove active class from all images
-    images.forEach(
-      // For every element in images array call a function that removes "active" class
-    )
+        images.forEach(element => {
+          element.classList.remove("active");
+        });
+
+        // indicators.forEach(el => {
+        //   el.classList.remove("active");
+        // });
+
+        while(indicators.length>0){
+          indicators[0].classList.remove("active");
+        }
+
+      images[index].classList.add("active");
+      indicators[index].classList.add("active");
+
+      images[index].classList.toggle("visible");
+    // For every element in images array call a function that removes "active" class
 
     // Remove active class from all indicators
     // Use loop to loop through all element of array
-
     // Add active class to the current image and indicator
     // images[]
+    console.log(index);
+    
   }
 
   // handle next click
   function showNextImage() {
     // increase currentIndex. What happens if the current index is > than number of images that we have?
     currentIndex++;
+    if (currentIndex>2){
+      currentIndex=0;
+    };
     showImage(currentIndex);
+    return currentIndex;
   }
 
   // handle previous click
   function showPrevImage() {
     // [image1, image2, image3]
     // decrease currentIndex. What happens if the current index is < 0?
+    currentIndex--;
+    if (currentIndex<0){
+      currentIndex=2;
```

Comment:
Do not hardcode currentIndex like this. It will not work if the number of images changes. Instead, set it to the `images.length -1`(position of the element of the image array)